feature_tests/cpp/include/ContiguousEnum.h

```diff
@@ -24,7 +24,6 @@ namespace capi {
 extern "C" {
 #endif
 
-void ContiguousEnum_destroy(ContiguousEnum* self);
```

Comment:
Question: why is there a change in `Cpp` output?

Reply:
We removed dtors from (non opaque) structs and enums, they're not useful.